cmd/sonobuoy/app/args.go

```diff
@@ -168,62 +172,48 @@ const (
 // users. Using e2e-parallel incorrectly has the potential to destroy clusters!
 func AddE2EConfigFlags(flags *pflag.FlagSet) *pflag.FlagSet {
 	e2eFlags := pflag.NewFlagSet("e2e", pflag.ExitOnError)
-	modeName := ops.NonDisruptiveConformance
-	defaultMode := modeName.Get()
-	e2eFlags.String(
-		e2eFocusFlag, defaultMode.E2EConfig.Focus,
-		"Specify the E2E_FOCUS flag to the conformance tests. Overrides --mode.",
-	)
-	e2eFlags.String(
-		e2eSkipFlag, defaultMode.E2EConfig.Skip,
-		"Specify the E2E_SKIP flag to the conformance tests. Overrides --mode.",
-	)
-	e2eFlags.String(
-		e2eParallelFlag, defaultMode.E2EConfig.Parallel,
-		"Specify the E2E_PARALLEL flag to the conformance tests. Overrides --mode.",
-	)
+
```

Comment:
The e2econfig type will eventually go away is the intent; but the registry config flag is very different so I'll do that in another PR